src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private ColorSupplier ballColor;
+    private Random ballNumber;
+
+    public ColorSupplier getBallColor() {
+        return ballColor;
+    }
+
+    public Random getBallNumber() {
+        return ballNumber;
+    }
+
+    public Ball getRandomBall() {
+        ballColor = new ColorSupplier();
+        ballNumber = new Random();
+        Ball ball = new Ball();
+        ball.setColor(ballColor.getRandomColor().name());
+        ball.setNumber(ballNumber.nextInt(100));
```

Comment:
Create constant for 100
```suggestion
        ball.setColor(colorSupplier.getRandomColor().name());
        ball.setNumber(random.nextInt(MAX_NUMBER));
```

Reply:
Created variable maxNumber as parameter in getRandomBall method.